integration/tests/basic/test_deposit.py

```diff
@@ -127,6 +130,7 @@ def test_transfer_spl_token_from_solana_to_neon(self, solana_account, pytestconf
 
         assert int(ata_balance_after.value.amount) == int(ata_balance_before.value.amount) + full_amount
 
+    @pytest.mark.mainnet
```

Comment:
tests with multipletokens mark wouldn't work on mainnet

Reply:
When we run tests with network == mainnet it will run only tests with mark "mainnet"
The logic you can see here https://github.com/neonlabsorg/neon-tests/blob/NDEV-2603-run-tests-on-mainnet/integration/tests/conftest.py#L51

Reply:
but now we have several tests with 2 marks: mainnet and multipletokens

Reply:
I did not include any tests with mark multipletokens for now cause we do not have multitokens feature in mainnet yet

Reply:
yep, I removed these 2 tests from mainnet run